README.md

```diff
@@ -1,3 +1,607 @@
+}).catch(handleErrorsFromHooks)
+```
+
+## APIs
+
+`@octokit/webhooks` provides 5 APIs that build upon each other.
+
+1. [**sign**](#sign)  
+   Create a signature based on a secret and webhook payload
+2. [**verify**](#verify)  
+   Verify a signature based on a secret and webhook payload
+3. [**Receiver**](#receiver)  
+   Provides events & hook APIs which are emitted based on webhook payloads you pass in.
+4. [**Middleware**](#middleware)  
+   Module that can be directly passed into a [Node HTTP Server](https://nodejs.org/api/http.html#http_class_http_server), [connect](https://github.com/senchalabs/connect) or [express](http://expressjs.com/).
+5. [**Server**](#server)  
+   Standalone server if all you need is receiving & handling webhooks without custom routes.
+
+Instances of `Receiver`, `Middleware` and `Server` expose [Events & Hook APIs](#events-and-hooks).
+
+### Sign
+
+```js
+const sign = require('@octokit/webhooks').sign
+// or: const sign = require('@octokit/webhooks/sign')
+
+const signature = sign({
+  data: webhookRequestBody,
+  secret: 'mysecret'
+})
+```
+
+`sign({data, secret})` returns a signature as String.
+
+<table width="100%">
+  <tr>
+    <th>
+      <code>
+        data
+      </code>
+    </th>
+    <td>
+      Object
+    </td>
+    <td>
+      <strong>Required.</strong>
+      Webhook request body as received from GitHub
+    </td>
+  </tr>
+  <tr>
+    <th>
+      <code>
+        secret
+      </code>
+    </th>
+    <td>
+      String
+    </td>
+    <td>
+      <strong>Required.</strong>
+      Secret as configured in GitHub Settings. If no <code>secret</code> was configured, don’t use the <code>sign()</code> method.
+    </td>
+  </tr>
+</table>
+
+### Verify
+
+```js
+const verify = require('@octokit/webhooks').verify
+// or: const verify = require('@octokit/webhooks/verify')
+
+const signatureMatches = verify({
+  data: webhookRequestBody,
+  secret: 'mysecret',
+  signature: 'sha1=0544bd735451ecd85a1297d2242ed9c38083aba7'
+})
+```
+
+`verify({data, secret, signature})` returns either `true` or `false`.
+
+<table width="100%">
+  <tr>
+    <th>
+      <code>
+        data
+      </code>
+    </th>
+    <td>
+      Object
+    </td>
+    <td>
+      <strong>Required.</strong>
+      Webhook request body as received from GitHub
+    </td>
+  </tr>
+  <tr>
+    <th>
+      <code>
+        secret
+      </code>
+    </th>
+    <td>
+      String
+    </td>
+    <td>
+      <strong>Required.</strong>
+      Secret as configured in GitHub Settings. If no <code>secret</code> was configured, don’t use the <code>verify()</code> method.
+    </td>
+  </tr>
+  <tr>
+    <th>
+      <code>
+        signature
+      </code>
+    </th>
+    <td>
+      String
+    </td>
+    <td>
+      <strong>Required.</strong>
+      Signature string as calculated by <code><a href="#sign">sign()</a></code>.
+    </td>
+  </tr>
+</table>
+
+### Receiver
+
+```js
+const createReceiver = require('@octokit/webhooks').receiver
```

Comment:
It'd be handy if `.receiver` was prefixed with `create` so it works with shorthand:

```js
const {createReceiver} = require('@octokit/webhooks')
```

Same goes for all the examples below.

Reply:
you can do `const createReceiver = require('@octokit/webhooks/receiver')` :) 

But yeah, I was thinking about that, too. I’m a friend of verbose methods, I’ll do the change